Share The Load/Model/Quiz.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Quiz.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 12/01/2021.
+//
+
+import Foundation
+
+final class Quiz: Codable {
```

Comment:
I don't think we'll need this codable unless we want it sent from the backend

Reply:
I think this object should be our next big task. Have a think about what it should do and we can discuss

Reply:
Removed codable, and will start to have a think about this object.